README.md

```diff
@@ -61,6 +61,9 @@ Astra Bulk Edit plugin can be used only with the Astra theme.
 
 ## Changelog ##
 
+### 2.0.0 ###
```

Comment:
```suggestion
### 1.2.7 ###
```

Reply:
No need to version it 2.0.0, as we are just adding compatibility for theme's major release.